package-lock.json

```diff
@@ -146,8 +146,8 @@
         "node": ">=22.8.0"
       },
       "optionalDependencies": {
-        "@esbuild/linux-arm64": "*",
-        "@esbuild/linux-x64": "*",
+        "@esbuild/linux-arm64": "latest",
```

Comment:
Remove from PR.

You can do so by checking out version from develop and committing that.

`git checkout develop -- package-lock.json`